packages/istanbul-lib-source-maps/lib/transform-utils.js

```diff
@@ -0,0 +1,20 @@
+/*
+ Copyright 2015, Yahoo Inc.
+ Copyrights licensed under the New BSD License. See the accompanying LICENSE file for terms.
+ */
+'use strict';
```

Comment:
I don't see this in some other files, we should probably be consistent.

Reply:
Added to the production sources of istanbul-lib-source-maps where it was missing.